src/browser/HostInstaller.h

```diff
@@ -34,7 +34,8 @@ class HostInstaller : public QObject
         CHROMIUM = 1,
         FIREFOX = 2,
         VIVALDI = 3,
-        TOR_BROWSER = 4
+        BRAVE = 4,
```

Comment:
Is it OK for the `BRAVE` entry to take `TOR_BROWSER`'s place in the list (so it's closer to Vivaldi), or should it be appended (in which case `TOR_BROWSER` would remain `4` and `BRAVE` would be `5`)? The former is what's in place here.

Reply:
Don't change the order. Also HostInstaller.cpp#L146 needs to be changed. The for loop is not pretty because it doesn't include the list size automatically.

Reply:
OK, changed. Done.